src/java/picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.java

```diff
@@ -267,7 +267,7 @@ public static void trackOpticalDuplicates(List<? extends ReadEnds> ends,
      * Looks through the set of reads and identifies how many of the duplicates are
      * in fact optical duplicates, and stores the data in the instance level histogram.
      */
-    private static void trackOpticalDuplicates(final List<? extends OpticalDuplicateFinder.PhysicalLocation> list,
+    protected static void trackOpticalDuplicates(final List<? extends OpticalDuplicateFinder.PhysicalLocation> list,
                                                final OpticalDuplicateFinder opticalDuplicateFinder,
```

Comment:
While you are here, perhaps add a comment about when one would use each version of trackOpticalDuplicates? I assume you plan to have your code call this version, as you are changing it to protected, but I am not sure why you want to do that, as you are setting orientationForOpticalDuplicates, which is only used by the other version of this function.